### PR TITLE
Added snap logo and fixed logo class

### PR DESCRIPTION
**Pull request by shivjoshi1996**
## Done

- Updated Snapcraft logo
- Updated logo class, making each logo span over 4 cols. 

## QA

Run demo and go down to "Who's using Vanilla" section.

## Issue / Card

Fixes: #133

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -151,17 +151,39 @@ <h3 class="p-heading-icon__title">Vanilla Design</h3>
...
+          <img class="p-inline-images__logo" src="https://assets.ubuntu.com/v1/adac6928-ubuntu.svg" alt="Ubuntu" />
+        </li>
+        <li class="p-inline-images__item">
+          <img class="p-inline-images__logo" src="https://assets.ubuntu.com/v1/4a93b95f-landscape_black-orange_hex.svg" alt="Landscape" />
+        </li>
+        <li class="p-inline-images__item">
+          <img class="p-inline-images__logo" src="https://assets.ubuntu.com/v1/4cd5caa2-maas_black-orange_hex.svg" alt="MAAS" />
+        </li>
+      </ul>
+      <ul class="p-inline-images u-no-margin--top">
```
Comment (Caleb-Ellis) [inline index.html]: You can just put everything under one `<ul>` rather than splitting into three

**Comment by shivjoshi1996** [comment]
@Caleb-Ellis Fixed

**Comment by shivjoshi1996** [comment]
@Caleb-Ellis Done

**Comment by kwm14** [comment]
## Visual review

Looking good @shivjoshi1996 just one small change.
- [ ] logos hidden from `$breakpoint-medium` through to `$breakpoint-x-small`

screen shot 2018-06-13 at 15 20 03

**Comment by shivjoshi1996** [comment]
@kwm14 Forgot to restart the demo, try again :100:

**Comment by kwm14** [comment]
Yaaaas. Looking good @shivjoshi1996 💯

**Comment by Caleb-Ellis** [comment]
Nice stuff, looking :100: @shivjoshi1996

**Pull request merged**